gulpfile.js

```diff
@@ -204,13 +205,13 @@ gulp.task('test-e2e', function(callback) {
   * */
   var testAppPath = path.resolve(__dirname, 'test/bare-bones-app');
   var testServerPath = require.resolve('./client/test/test-server');
-  var testServerOpts = {cwd: __dirname};
+  var testServerOpts = {cwd: path.join(__dirname, '/client/test/sandbox')};
```

Comment:
This path doesn't exist when this is run, so `spawn()` can't cd into it to run the test server.